docs/hugo/content/introduction/multitenant-deployment.md

```diff
@@ -3,16 +3,48 @@ title: Deploying Azure Service Operator v2 in multi-tenant mode
 linktitle: Multitenancy
 ---
 
-The current release version of the Azure Service Operator (ASO) deployment YAML installs the operator in _single-tenant_ mode:
-deployed in the `azureserviceoperator-system` namespace with _a single set of Azure credentials_,
-and manages resources in _any namespace_ in the cluster.
-That single operator deployment handles webhooks fired when Azure resources are changed.
+Currently, we support two types of multitenancy with Azure Service Operator (ASO): 
```

Comment:
Don't make readers scan down the document to find the second type of multitenancy. 

Either just give their names:

```suggestion
Currently, we support two types of multitenancy with Azure Service Operator (ASO): single operator and multiple operator.
```

or give one-line definitions:

```suggestion
Currently, we support two types of multitenancy with Azure Service Operator (ASO): 
* _single operator_ where a single ASO instance uses different credentials for different resources; and
* _multiple operator_ where an individual ASO instance is configured for each Kubernetes namespace.
```